src/game/construct_types.h

```diff
@@ -16,5 +16,10 @@ struct ConstructData {
 namespace Constructs {
     const static ConstructData CHEST = { ConstructType::CHEST, 50, "Chest", "A treasure chest full of gold.", "test.png" };
     const static ConstructData SPIKES = { ConstructType::SPIKES, 50, "Spikes", "It was me, spikes.", "dio.jpg" };
-    const static ConstructData REMOVE = { ConstructType::REMOVE, 0, "Remove", "Remove a block.", "Tomato.jpg" };
+    const static ConstructData REMOVE = { ConstructType::REMOVE, 200, "Remove", "Remove a block.", "Tomato.jpg" };
+    const static std::map<ConstructType, ConstructData> CONSTRUCTS = {
```

Comment:
Hmm, the intention was to provide a way to get the `ConstructData` corresponding to a `ConstructType` enum without having a reference to the `Construct`.  This is useful when we're passing `ConstructType`s around events, and especially since we only get the construct type as an int over the network.  What do you suggest?